ocs_ci/deployment/deployment.py

```diff
@@ -990,7 +995,7 @@ def setup_local_storage(storageclass):
         f"{constants.OPERATOR_NODE_LABEL}"
     )
 
-    ocp_version = get_ocp_version()
+    ocp_version = float(get_ocp_version())
```

Comment:
This is problematic. As on line 1087 you comparing string and float now.
https://github.com/red-hat-storage/ocs-ci/pull/3221/files#diff-8571fc52f1040ac2cb01985da6d61cc93a535f812814c200b45d00daf28c2e3fR1087

So either do not change it here, or change all usage for compare with this variable.

Reply:
Sorry looks like i changed the wrong line